frontend/src/components/dashboard/minesHomePage/MineSearch.js

```diff
@@ -38,7 +38,7 @@ const defaultProps = {
 };
 
 const checkAdvancedSearch = ({ status, region, tenure, commodity, tsf, major }) =>
-  tsf || major || some([status, region, tenure, commodity], negate(isEmpty));
+  tsf || major || some([status, region, tenure, commodity, tsf], negate(isEmpty));
```

Comment:
I don't think this change is necessary with the incorrect `tsf: []` initialValues removed

Reply:
removed